.github/release.yml

```diff
@@ -0,0 +1,7 @@
+changelog:
+  exclude:
+    labels:
+      - internal
+    authors:
+      - dependabot
+      - github-actions
```

Comment:
Is this an OR condition or AND?

Will it only exclude that label when it's from those authors?

Reply:
It's OR

Reply:
At least, I think it is!

Reply:
Yes it's OR. I have seen it ignore dependabot PRs on adminlte repo, and those were not labelled, too.